environment.yml

```diff
@@ -36,8 +36,9 @@ dependencies:
   - psutil
   - rmg::pydas >=1.0.3
   - pydot
+  - conda-forge::julia=1.8.5
+  - conda-forge::suitesparse
   - rmg::pydqed >=1.0.3
-  - rmg::pyjulia
```

Comment:
Is pyjulia installed through `conda-forge::julia=1.8.5`?

Reply:
It's not, and the rmg:pyjulia does not either. The conda-recipes repo has the build script and you can see that it only provides Julia (as far as I can tell, I'm new to building conda packages). I think pyjulia is an implicit dependency of something else in the environments file? So it is added when the environment is solved?

Reply:
So, correct if I am misunderstanding what you are saying, but we will need to keep pyjulia as it is the connection between Julia and Python. So I guess we can either keep getting from the rmg channel or from conda-forge channel.

However, the CI appears to be passing? I am a bit confused though how it is doing that when pyjulia isn't installed in the rmg_env (I am assuming here or really misunderstanding something)

Reply:
> So, correct if I am misunderstanding what you are saying, but we will need to keep pyjulia as it is the connection between Julia and Python. So I guess we can either keep getting from the rmg channel or from conda-forge channel.
> 
> However, the CI appears to be passing? I am a bit confused though how it is doing that when pyjulia isn't installed in the rmg_env (I am assuming here or really misunderstanding something)

My mistake - conda still ends up retrieving pyjulia from the rmg channel. See [this line in the latest CI run](https://github.com/ReactionMechanismGenerator/RMG-Py/actions/runs/4953296609/jobs/8860637395#step:4:321).

CC'ing @mjohnson - pyjulia, how does it work? The conda-recipe doesn't make sense to us.

Reply:
So my understanding of PyJulia is that, assuming you have Julia installed and set in the PATH (or set in our conda env path for our case when rmg_env is active), it is imported in python like so:

```python
import julia
```

Then we use PyJulia for the `julia.install()` which does the whole PyCall adding (if not present) and building (unless it is M1, then it appears to have an issue with that

We can also use PyJulia to create our system images for precompiling

Reply:
Apologies if that was already understood

Reply:
It is a dependency of pyrms so including pyrms in the environment.yml file will cause it to be installed as well.